source/edutravel/travel_agency/models.py

```diff
@@ -20,6 +20,11 @@ def __str__(self):
 
 class Employee(Person):
     '''Employee table'''
+    first_name = models.CharField(max_length=80)
+    last_name = models.CharField(max_length=80)
+    phone = models.IntegerField(max_length=12)
+    email = models.EmailField(max_length=50)
+    passport_number = models.IntegerField(max_length=11)
```

Comment:
The Employee inherits from Person. In this case since the attributes are the same with the exception of max_length being different we can add those to the Person model instead.